docs/monitors/collectd-postgresql.md

```diff
@@ -39,15 +40,15 @@ monitors:
     - "hostname"
     statement: "Select * From test Where host = $1;"
     results:
-    - type: gauge
+    - type: "gauge"
```

Comment:
It's fine to leave them in, but you typically don't need quotes in YAML unless the string contains special characters.